lib/social.ts

```diff
@@ -0,0 +1,6 @@
+import { Story } from '@prisma/client'
+import { storyCite } from '../components/stories/model'
+
+export default function generateSocial(story: Story, contentSize = 150) {
```

Comment:
content size is missing a type on it :)

Reply:
@JamiesonRoberts The type is inferred by the default. ESLint will throw an error if you try to type them (https://palantir.github.io/tslint/rules/no-inferrable-types/)